### PR TITLE
Ido fix/post audit

### PR DESCRIPTION
**Pull request by idirall22**
- Clean Node Operator Contract

### CODE REVIEW

File: contracts/StMATIC.sol [hunk 1]
```diff
@@ -382,17 +382,7 @@ contract StMATIC is
...
-                operatorInfos[i].validatorShare
-            );
-            uint256 stMaticReward = validatorShare.getLiquidRewards(
-                address(this)
-            );
-            uint256 rewardThreshold = validatorShare.minAmount();
-
-            if (stMaticReward < rewardThreshold) {
-                validatorShare.withdrawRewards();
-            }
```
Comment (GitMark0) [inline contracts/StMATIC.sol:395]: Why is this removed? This prevents distributeRewards to fail if liquid reward is less than minAmount.

**Review by GitMark0 — CHANGES_REQUESTED** [review]

File: contracts/StMATIC.sol (hunk 1 above)
Comment (idirall22) [reply contracts/StMATIC.sol:395]: I moved the logic inside the NodeOperatorContract
https://github.com/Shard-Labs/PoLido/blob/ido-fix/post-audit/contracts/NodeOperatorRegistry.sol#L959
